helm/jupyterhub/files/dandi_authenticator.py

```diff
@@ -0,0 +1,77 @@
+from kubernetes_asyncio import client
```

Comment:
is this file done from scratch or based on some already existing code in dandi-archive or hub? if so -- please add reference here

Reply:
This from https://github.com/dandi/dandi-hub/blob/dandi/config.yaml.j2

In the jupyterhub config, `extraConfig` is executed as python. I just moved it into a python file.